TestApp/android/app/src/main/java/com/testapp/TestAppNativeModule.java

```diff
@@ -32,6 +33,8 @@
 
     private static final String START_AUTOMATICALLY = "start_automatically";
 
+    private static final String SESSION_TRACKER_ENABLED_KEY = "session_tracker_enabled";
```

Comment:
let's call it `MANUAL_SESSION_TRACKER_ENABLED_KEY`, and the same for the string value.

Reply:
also let's remove empty line between two static fields.